specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/webapplicationfirewall.json

```diff
@@ -329,6 +329,10 @@
             "name": "WebApplicationFirewallPolicyResourceState",
             "modelAsString": true
           }
+        },
+        "botDetection": {
```

Comment:
We probably need a global enable/disable switch

Reply:
good point